doc/nrf/releases/release-notes-2.1.0.rst

```diff
@@ -0,0 +1,93 @@
+* Added experimental support for feature complete Matter over Thread including :ref:`Thread v1.3 certified stack <thread_ug_supported_features_v13>`.
+* Added experimental support for setting the TX power envelope for Bluetooth® LE and IEEE 802.15.4.
+* Added support for enhanced power optimized device management and nRF Cloud Location Services through LwM2M/CoAP with AVSystem.
+* Updated Trusted Firmware-M to v1.6, which brings flash and RAM overhead optimizations and replaces Secure Partition Manager (SPM) from this release onwards.
+
+See :ref:`ncs_release_notes_210_changelog` for the complete list of changes.
+
+Sign up for the `nRF Connect SDK v2.1.0 webinar`_ to learn more about the new features.
+
+The official nRF Connect for VS Code extension also received improvements for this release, including a customized debugger experience.
+See the `nRF Connect for Visual Studio Code`_ page for more information.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.1.0**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.1.0`_.
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the only officially supported IDE for |NCS| v2.1.0.
+SEGGER Embedded Studio Nordic Edition is no longer tested or recommended for new projects.
+
+:ref:`gs_app_tcm`, used to install the |NCS| automatically from `nRF Connect for Desktop`_, is available for Windows, Linux, and macOS.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.1.0_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.1.0_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.1.0`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_210_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+* PENDING UPDATE
```

Comment:
```suggestion
* Updated:

  *  :ref:`mcuboot:readme-ncs.rst` now includes information regarding the bootloader user guides in the |NCS| documentation.

* Removed:

  * The *Zephyr Test Plan* page in the MCUboot documentation set.
  * The *Building and using MCUboot with Zephyr* page in the MCUboot documentation set.
```

If the ref to the mcuboot docset doesn't work (it might not be implemented), this is the direct link to that page:  https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/mcuboot/readme-ncs.html

Reply:
Applied, please re-check @FrancescoSer .